docs/provisioning/external-silos.md

```diff
@@ -0,0 +1,154 @@
+  - For creating a k8s cluster on-premises one can use [Kind](https://kind.sigs.k8s.io/), for instance.
+  - For creating a k8s cluster in Azure (in a different tenant otherwise we'd be dealing with _internal_ silos), one can use [Azure Kubernetes Service (AKS)](https://portal.azure.com/#create/microsoft.aks).
+- FL Admin needs to have **Owner** role in the subscription, or at least in the resource group where the Azure ML workspace will be created. This is because some Role Assignments will have to be created.
+- Silo Admin must be given (temporary) Contributor role to the subscription, or at least the [Azure Arc Onboarding](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#kubernetes-cluster---azure-arc-onboarding) built-in role. This is because one step will require access to both the orchestrator subscription, and to the k8s cluster. It assumed that _the FL Admin shouldn't have direct access to the k8s cluster_.
+
+
+## Procedure
+
+### A. Create the Azure ML workspace and the orchestrator
+
+> This is all explained in the first sections of the [cookbook](./README.md) but repeated here for convenience and to clarify that it needs to be done by the **FL Admin**.
+
+**FL Admin** first creates an open Azure ML workspace named `<workspace-name>`. FL Admin will need Owner permissions in `<workspace-resource-group>`, since Role Assignments will need to be created later on. (The `<base-name>` value will be used when creating associated resources and can be chosen arbitrarily, but note that it should be unique in the subscription.)
+```bash 
+az deployment group create --template-file ./mlops/bicep/modules/azureml/open_azureml_workspace.bicep --resource-group <workspace-resource-group> --parameters machineLearningName=<workspace-name> baseName=<base-name>
+```
+After that, **FL Admin** creates the compute and storage corresponding to the orchestrator (the value of the `pairBaseName` parameter will need to be adjusted if you have already created an orchestrator with this base name in the subscription).
+```bash
+az deployment group create --template-file ./mlops/bicep/modules/fl_pairs/open_compute_storage_pair.bicep --resource-group <workspace-resource-group> --parameters pairBaseName="orch" machineLearningName=<workspace-name>
+```
+
+### B. Connect the existing k8s cluster to Azure Arc
+Detailed instructions for this phase, including steps for verification or for slightly different use cases can be found [there](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli). Here is a summary, which should be all you need.
+
+0. In addition to the prerequisites listed [above](#prerequisites), make sure you have the following:
+    - an identity that can be used to log in to the Azure CLI and connect the k8s cluster to Azure Arc;
+    - [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) with version >= 2.24.0;
```

Comment:
Could we move this into the requirements then?

Or you could create a section "meet the requirements" at the beginning where you provide those instructions first.

Reply:
moved into the Prerequisites section, and clarified which of the Silo/FL Admin needed what.